### PR TITLE
Add configurable prefix for S3 bucket

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Configurable prefix for S3 destination bucket.
Using the INSIGHTS_RESULTS_AGGREGATOR_EXPORTER__S3__PREFIX environment variable, we can define an output path with the {PREFIX}/_metadata.csv format

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

- Updated UTs

## Checklist
* [x] `make before_commit` passes
* [x] updated documentation wherever necessary
* [x] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: exporter.go [hunk 1]
```diff
@@ -222,7 +223,8 @@ func performDataExportToS3(configuration *ConfigStruct,
 	// log into terminal
 	printTables(tableNames)
 
-	bucket := GetS3Configuration(configuration).Bucket
+	bucket := GetS3Configuration(configuration).Prefix + "/" + GetS3Configuration(configuration).Bucket
```
Comment (tisnik) [inline exporter.go:226]: WDYT about ability to have empty prefix, i.e. bucket path w/o `/` at the beginning?

**Review by tisnik — APPROVED** [review]
ok

File: exporter.go (hunk 1 above)
Comment (epapbak) [reply exporter.go:226]: I thought we wouldn't want that, but since you're asking... that's a quick to make change =)

Comment (epapbak) [reply exporter.go]: done

**Review by juandspy — APPROVED** [review]

**Pull request merged**